frontend/src/streamlist/__tests__/StreamList.test.jsx

```diff
@@ -80,23 +83,23 @@ test("Clicking dropdown on table row displays additional information", () => {
   });
 
   const dropdownButton = document.querySelectorAll(
-    "td.dropdownButton button"
+    "tr.MuiTableRow-root[index] button"
   )[0];
   act(() => {
     ReactTestUtils.Simulate.click(dropdownButton);
   });
 
   const additionalInfoElement = document.querySelector(
-    "tr.streamDetails td div div div div span.MuiTypography-caption"
+    "tr.MuiTableRow-root td.MuiTableCell-root[colspan] h6"
   );
   expect(additionalInfoElement).not.toBe(null);
-  expect(additionalInfoElement.innerHTML).toBe(
-    "Additional Stream Details go here"
-  );
+  expect(additionalInfoElement.innerHTML).toBe("Stream Details");
```

Comment:
should we make an issue for the fact that our current creation & viewing setup for streams doesn't actually let us give streams any additional details?

anyway this is where my previous comment about `streamDetails` instead of `dataSource` finds its root confusion. stream details are their own thing

Reply:
Yeah the prop name should probably be changed as you suggested. When you say that streams don't have any additional details, do you mean with how streams are currently created there aren't any additional details?

Reply:
Yeah the prop name should probably be changed as you suggested. When you say that streams don't have any additional details, do you mean with how streams are currently created there aren't any additional details?

Reply:
yeah. there's no field to add them on creation and no way to edit a stream to add it. i don't see any issues already in our backlog for stream modification or details or anything so...

Reply:
I think it's just there as a placeholder for now in case it's needed in the future. I think it's okay to leave it be until we are completely sure we don't need it since it's easy to add/remove using the library.